src/shared/onboarding.ts

```diff
@@ -17,12 +19,37 @@ export const educationLevels = [
 ];
 
 export const genders = [
-  "Masculino",
+  "Agênero",
   "Feminino",
-  "Não-binário",
   "Gênero fluido",
-  "Agênero",
+  "Masculino",
+  "Não-binário",
   "Transgênero",
+  "Outros",
+  "Prefiro não responder",
 ];
 
-genders.sort();
+export const formSchema = z.object({
+  nomeSocial: z
+    .string()
+    .nonempty("Nome Social é obrigatório")
+    .min(3, "Mínimo 2 caracteres"),
+  idade: z
+    .string()
+    .nonempty("Idade é obrigatório")
+    .max(2, "Insira idade válida"),
+  telefone: z.string().nonempty("Telefone é obrigatório"),
+  pais: z.string().nonempty("País é obrigatório"),
+  cidadeEstado: z.string().nonempty("Cidade é obrigatório"),
+  email: z
+    .string()
+    .nonempty("Email é obrigatório")
+    .email("Insira um email válido"),
+  profissao: z.string(),
+  empresaOrganizacao: z.string(),
+  githubPortifolio: z.string(),
+  linkedin: z.string(),
+  gender: z.string().nonempty("Gênero é obrigatório"),
+  qOne: z.string().nonempty("Resposta é obrigatório"),
+  qTwo: z.string().nonempty("Resposta é obrigatório"),
+});
```

Comment:
This looks good, but we should consider breaking the for down into multiple steps, to avoid making a huge form

Reply:
you can break it down then merge it back before posting. [here](https://zod.dev/?id=merge)

Reply:
Great As discussed with Marco, we gonna do this in a next PR